components/sections/faqs/variant_a.tsx

```diff
@@ -26,20 +31,90 @@ function VariantA({ subtitle, title, faqs }: FAQProps) {
         (items) => items?.question?.toLowerCase().includes(searchTerm)
       ); // get search results based on data
 
-  // display FAQs
-  const FAQs = ({ items }) => {
-    return (
-      <ul className="space-y-4 lg:space-y-6">
-        {items?.map((faq, index) => (
-          <li className="rounded bg-gray-50 p-6 shadow" key={index}>
-            <button
-              aria-label={faq?.question}
-              className="font-heading flex w-full items-center justify-between border-none text-left font-bold hover:text-gray-600 focus:outline-none"
+  // change page
+  const changePage = (buttonNumber) => setCurrentPage(buttonNumber);
+
+  return (
+    <section className="py-20 bg-gray-50">
+      <Container>
+        <Container maxWidth={576} className="mb-16 text-center ">
+          <Text weight="bold" className="text-primary">
```

Comment:
Kindly update the Subtitle and Title elements to be returned only when it is defined/there is a value.